README.md

```diff
@@ -9,14 +9,14 @@ Hyperledger Burrow is a permissioned Ethereum smart-contract blockchain node bui
 
 ## Table of Contents
 
-- [What is burrow](#what-is-burrow)
+- [What is Burrow ?](#what-is-burrow-)
 - [Installation](#installation)
-- [For developers](#for-developers)
+- [For Developers](#for-developers)
```

Comment:
We are using drop case on other words elsewhere, omit this and I will accept

Reply:
Do you want to undo, otherwise delete "[For developers](#for-developers)"?

Reply:
@qpakzk see my comment below, and PR #700